lerna.json

```diff
@@ -1,5 +1,8 @@
 {
   "version": "independent",
   "npmClient": "npm",
-  "packages": ["packages/*"]
+  "packages": ["packages/*"],
+  "nohoist": [
+    "@deriv/api-types"
+  ]
```

Comment:
@hamid-deriv We need to leave a todo here that we should revert this later, Otherwise it will be forgotten and will become a thing.
We cannot leave a todo in JSON file, Any idea what to do?

Reply:
@hamid-deriv Could you update the dependency in all packages instead and remove this from here🙏